File: C#/C# Advanced/StringsTextProcessing/03_CorrectBrackets/CorrectBrackets.cs
using System;

class CorrectBrackets
{
    static void Main()
    {
        string input = Console.ReadLine();

        if (input[0] == ')')
        {
            Console.WriteLine("Incorrect");
            return;
        }

        int leftCount = 0;
        int rightCount = 0;

        foreach (var item in input)
        {
            if (item == ')')
            {
                rightCount++;
            }
            else if (item == '(')
            {
                leftCount++;
            }
        }

        if (leftCount == rightCount)
        {
            Console.WriteLine("Correct");
        }
        else
        {
            Console.WriteLine("Incorrect");
        }
    }
}

File: Javascript/Javascript Fundamentals/Strings/correct-brackets.js
function solve(args) {
    var input = args[0];

    if (input[0] == ')')
    {
        console.log("Incorrect");
        return;
    }

    var leftCount = 0;
    var rightCount = 0;

    for (var item of input)
    {
        if (item == ')')
        {
            rightCount += 1;
        }
        else if (item == '(')
        {
            leftCount += 1;
        }
    }

    if (leftCount == rightCount)
    {
        console.log("Correct");
    }
    else
    {
        console.log("Incorrect");
    }
}